lib/sentry/application.ex

```diff
@@ -48,16 +52,18 @@ defmodule Sentry.Application do
   end
 
   defp start_integrations(config) do
-    if config[:oban][:cron][:enabled] do
-      Sentry.Integrations.Oban.Cron.attach_telemetry_handler()
-    end
+    if Code.ensure_loaded?(Sentry.Integrations.CheckInIDMappings) do
```

Comment:
Wait, why would this not be loaded? This will always be loaded. We can also always start it even if integrations are not started for the time being, it's very cheap to run a GenServer 🙃

Reply:
Do you think it's unnecessary to make sure the table was created?

Reply:
1. `:ets.new/2` doesn't really fail, so the table will pretty much always be created.
2. That's not what `Code.ensure_loaded?/1` does: it only checks that the bytecode for the module is loaded in the VM and that the module is available, **not** that the GenServer was successfully started.
3. If the supervisor started without issues, then by definition the GenServer also started (at least the first time) without issues.